ISISComputingGroup/EPICS-ioc
# Ticket4555: SANS2D Vacuum System

### Description of work

Should cover:
* https://github.com/ISISComputingGroup/IBEX/issues/4555
* https://github.com/ISISComputingGroup/IBEX/issues/4556
* https://github.com/ISISComputingGroup/IBEX/issues/4557
* https://github.com/ISISComputingGroup/IBEX/issues/4558
* https://github.com/ISISComputingGroup/IBEX/issues/4560
* https://github.com/ISISComputingGroup/IBEX/issues/4561
* https://github.com/ISISComputingGroup/IBEX/issues/4562
* https://github.com/ISISComputingGroup/IBEX/issues/4563
* https://github.com/ISISComputingGroup/IBEX/issues/4564
* https://github.com/ISISComputingGroup/IBEX/issues/4565

Tick these off on https://github.com/ISISComputingGroup/IBEX/issues/4554 once done.

See documentation at https://github.com/ISISComputingGroup/ibex_developers_manual/wiki/PLCs

System tests are hard, see https://github.com/ISISComputingGroup/IBEX/issues/5403

### To test

* Connect to the FINS PLC on SANS2D (ask me for the IP address)
* Confirm that it shows the same information as the VI running on SANS2D

---

#### Code Review

    - [PV naming](https://github.com/ISISComputingGroup/ibex_developers_manual/wiki/PV-Naming).
    - [Disable records](https://github.com/ISISComputingGroup/ibex_developers_manual/wiki/Disable-records)
    - [Record simulation](https://github.com/ISISComputingGroup/ibex_developers_manual/wiki/Record-Simulation)
    - [Finishing touches](https://github.com/ISISComputingGroup/ibex_developers_manual/wiki/IOC-Finishing-Touches)

### Functional Tests

- IOC responds correctly in:

### Final steps
